Fix compilation error bug #279143, due to missing libgnomeui CFLAGS.
Fix another bug error during the link phase due to missing libgnomeui LIBS.

 Makefile.am  |    4 ++--
 configure.ac |    3 +++
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 3de3b2b..0f18f44 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,9 +6,9 @@ dist_servers_DATA = GNOME_Swallow.server
 
 panel_PROGRAMS = gnome_swallow
 
-INCLUDES = ${PANEL_CFLAGS}
+INCLUDES = ${PANEL_CFLAGS} ${LIBGNOMEUI_CFLAGS}
 
 gnome_swallow_SOURCES = gnome-swallow.c
-gnome_swallow_LDADD = ${PANEL_LIBS} 
+gnome_swallow_LDADD = ${PANEL_LIBS} ${LIBGNOMEUI_LIBS}
 
 EXTRA_DIST = debian/dirs debian/control debian/changelog debian/copyright debian/install debian/rules
diff --git a/configure.ac b/configure.ac
index c20f63e..d9bccb5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,8 +9,11 @@ AC_PROG_CC
 
 PKG_PROG_PKG_CONFIG
 PKG_CHECK_MODULES(PANEL, [libpanelapplet-2.0])
+PKG_CHECK_MODULES(LIBGNOMEUI, [libgnomeui-2.0])
 AC_SUBST(PANEL_CFLAGS)
 AC_SUBST(PANEL_LIBS)
+AC_SUBST(LIBGNOMEUI_CFLAGS)
+AC_SUBST(LIBGNOMEUI_LIBS)
 
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
